frontend/package.json

```diff
@@ -32,7 +32,9 @@
     "@web3-onboard/walletconnect": "^2.1.3",
     "bnc-notify": "^1.5.1",
     "core-js": "^3.6.5",
+    "ethereum-cryptography": "^1.1.2",
```

Comment:
_(arbitrary comment location to start a thread)_

> Does the homepage look okay with 4 boxes? It looks a little wonky to me, but I am wondering what other people think.

I agree it looks a bit wonky. Wdyt about putting the send history in the account info box, similar to what uniswap does? We can basically put the table component there, may need some formatting changes though if it doesn't fit nicely

<img width="323" alt="image" src="https://user-images.githubusercontent.com/17163988/204658062-8a7becc1-d074-4825-9ad9-be96fb2ff902.png">

<img width="372" alt="image" src="https://user-images.githubusercontent.com/17163988/204658180-2c431fb5-244c-4d44-9ea2-5ffcd13419f8.png">

<img width="368" alt="image" src="https://user-images.githubusercontent.com/17163988/204658246-15478de4-7460-4be7-941f-ab9ada5e4cc8.png">

Reply:
Yes, totally I can give that a shot